### PR TITLE
WIP: Add documentation for CRI API moving to v1beta1

### PR DESCRIPTION
**Pull request by mrunalp**
This is a placeholder PR for this enhancement: https://github.com/kubernetes/enhancements/issues/2040

Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: content/en/docs/concepts/containers/container-runtime-interface.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+
```
Comment (sftim) [inline content/en/docs/concepts/containers/container-runtime-interface.md:1]:
```suggestion
```
(remove the leading blank line)

File: content/en/docs/concepts/containers/container-runtime-interface.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+- dchen1107
+- derekwaynecarr
+title: Container Runtime Interface
+content_type: concept
+weight: 20
+---
+
+<!-- overview -->
+
+{{< feature-state for_k8s_version="v1.20" state="beta" >}}
```
Comment (sftim) [inline content/en/docs/concepts/containers/container-runtime-interface.md:13]: What does this represent? Is this that Kubernetes supports the CRI, as a beta Kubernetes feature? Or that Kubernetes supports using [some set of versions of] the CRI API?

Maybe the CRI support in Kubernetes is beta _and_ CRI itself is beta?

**Review by sftim — COMMENTED** [review]
https://kubernetes.io/docs/concepts/extend-kubernetes/compute-storage-net/device-plugins/ documents a different kind of plugin API and might be useful as something to base the page upon.

Consider splitting the information added here into:
- a new concept page
- a more detailed page inside https://kubernetes.io/docs/reference/

For the full PR, consider revising https://kubernetes.io/docs/setup/production-environment/container-runtimes/ either with links or with new text.

**Comment by sftim** [comment]
Hi @mrunalp 

What help would you like towards getting this PR ready for review? Here in SIG Docs we'd really appreciate a chance to get as much reviewing done as we can before KubeCon.

**Comment by irvifa** [comment]
Hi @mrunalp since this is been merged kubernetes/kubernetes#96387  and the rest will be on 1.21 wondering if there's something we could help so that we can proceed with this placeholder?

**Comment by mrunalp** [comment]
@irvifa, we will have to hold this till 1.21. We don't need doc updates in 1.20.

**Comment by annajung** [comment]
> @irvifa, we will have to hold this till 1.21. We don't need doc updates in 1.20.

Since this is no longer required, I'll close this PR. Feel free to reopen when the next release milestone and branch is available

/milestone clear
/close

**Pull request closed without merging**